Project.toml

```diff
@@ -13,6 +13,7 @@ DataValueInterfaces = "e2d170a0-9d28-54be-80f0-106bbe20a464"
 
 [extras]
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
```

Comment:
Why do we need `Compat` here?

Reply:
This is for using `hasproperty` in test.